### DISCUSSION TITLE
Can I use codesandbox client in my own react projects?

### DISCUSSION THREAD

**Discussion by ghost** [codesandbox/codesandbox-client#6069] [Q&A] [answered]
I can see that codesandbox client is open source.

It's not clear to me if this is intended to allow me to integrate it into my own projects?

If so, is there any documentation that describes how to do this?  I can't seem to see anything.

thanks!

**Comment by smhutch** [comment, marked as answer]
Hi @bootrino, you might be interested in using our [Sandpack](https://github.com/codesandbox/sandpack) library to integrate aspects of the CodeSandbox experience into your own React projects.

**Reply by Pixele9** [reply]
Is there a way to show the file system using Sandpack?

**Reply by danilowoz** [reply]
Hey @Pixele9.Very soon, Sandpack will officially export a `SandpackFileExplorer` component, but meanwhile, you can pick the file-explorer component from this sandbox https://codesandbox.io/s/file-explorer-1ffn6 

Hope it helps you

**Reply by Pixele9** [reply]
Great news! Looking forward on that `SandpackFileExplorer` component Thank you very much @danilowoz

**Reply by Lilsimonsky** [reply]
Hi bro I'm new here, I really wanna be on the same field with y'all can someone help
